test(transformer): remove dead code from fixtures update script

overlookmotel opened the pull request:
Follow-up after #7771 and #7779. Remove dead code from transformer fixtures updater script.

overlookmotel commented:
* **#7808** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/7808?utm_source=stack-comment-icon" target="_blank"><img src="https://static.graphite.dev/graphite-32x32-black.png" alt="Graphite" width="10px" height="10px"/></a> 👈 <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/7808?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.
<!-- Current dependencies on/for this PR: -->

Dunqing commented:
### Merge activity

* **Dec 12, 4:27 AM EST**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

overlookmotel committed:
test(transformer): remove dead code from fixtures update script (#7808)

Follow-up after #7771 and #7779. Remove dead code from transformer fixtures updater script.
